AndroidAnnotations/androidannotations/src/main/java/org/androidannotations/handler/AbstractPreferenceListenerHandler.java

```diff
@@ -58,6 +71,28 @@ protected final Res getResourceType() {
 		return Res.STRING;
 	}
 
+	protected final boolean isPreferenceOrSubclass(TypeMirror elementType) {
+		String type = elementType.toString();
+		if (type.equals(CanonicalNameConstants.PREFERENCE)) {
```

Comment:
I do not think this is necessary. The `isSubtype()` [JavaDoc](https://docs.oracle.com/javase/7/docs/api/javax/lang/model/util/Types.html#isSubtype%28javax.lang.model.type.TypeMirror,%20javax.lang.model.type.TypeMirror%29) says: 

>  Any type is considered to be a subtype of itself.